packages/app/src/debug/DebugFailedTest.cy.tsx

```diff
@@ -197,6 +197,9 @@ describe('<DebugFailedTest/>', () => {
 
     assertRowContents(testResult)
 
+    cy.contains('...').realHover()
+    cy.contains('[data-cy=tooltip-content]', 'Test content 2').should('be.visible')
```

Comment:
This should contain everything that was hidden by the ellipsis separated by `>`
```suggestion
    cy.contains('[data-cy=tooltip-content]', 'Test content 2 > Test content 3 > Test content 4').should('be.visible')
```